dwitter/templates/dweet/dweet.html

```diff
@@ -209,9 +212,15 @@
   
     var time = 0;
     var frame = 0;
+    
     function u(t) {
-       {{ code | safe }}
-      }
+      {{ code | safe }}
+    }
+    
+    
+    let src = (u + "").split(/[{}]/g)[1];
```

Comment:
Haven't had the chance to play with this, but couldn't you craft a dweet that would break this?